pkg/api/v1/types.go

```diff
@@ -780,6 +782,8 @@ const (
 	MigrationSucceeded VirtualMachineInstanceMigrationPhase = "Succeeded"
 	// The migration failed
 	MigrationFailed VirtualMachineInstanceMigrationPhase = "Failed"
+	// The migration is aborting
+	MigrationAborting VirtualMachineInstanceMigrationPhase = "Aborting"
```

Comment:
Please use a condition for that. See: #1611.

Reply:
Ther constant is probably unused now?

Reply:
ah, yea, I missed that. Will remove it.